frontend/lib/rental-history.tsx

```diff
@@ -8,22 +8,39 @@ import { TextualFormField } from './form-fields';
 import { SessionUpdatingFormSubmitter } from './session-updating-form-submitter';
 import { RhFormMutation, BlankRhFormInput } from './queries/RhFormMutation';
 import { exactSubsetOrDefault } from './util';
-import { NextButton, BackButton } from './buttons';
+import { NextButton, BackButton, CenteredPrimaryButtonLink } from './buttons';
 import { PhoneNumberFormField } from './phone-number-form-field';
 import { AppContext } from './app-context';
+import { Link } from 'react-router-dom';
 
 const RH_ICON = "frontend/img/ddo/rent.svg";
 
 /* Rental history welcome page */
 
 function RentalHistoryWelcome(): JSX.Element {
     return (
-      <Page title="Request your rental history through our online form" withHeading="big" className="content">
-        <div className="content">
-          <StaticImage src={RH_ICON} alt="rent-icon" ratio="is-128x128" />
-        </div>
-        <p>Let's help you request your <b>rental history</b>! This document, kept by the NY Division of Housing & Community Renewal (DHCR), helps you find out if you're being overcharged.</p>
-        <p><em>This service is free, secure, and confidential.</em></p>
+      <Page title="Request your rental history">
+         <section className="hero is-light">
+          <div className="hero-body">
+            <div className="has-text-centered">
+              <div className="is-inline-block jf-rh-icon">
+                <StaticImage ratio="is-square" src={RH_ICON} alt="" />
+              </div>
+              <h1 className="title is-spaced">
+                Request your rental history in two simple steps!
+              </h1>
+              <p className="subtitle">
+                Let's help you request your <b>rental history</b>! This document, kept by the NY Division of Housing & Community Renewal (DHCR), helps you find out if you're being overcharged.
```

Comment:
Does the `&` here look like an actual ampersand on the page?  GitHub's syntax highlighting seems to think it's an error--possibly we should be using `&amp;` instead.

Reply:
all fixed!